packages/validator/test/cli-validator/tests/option-handling.test.js

```diff
@@ -190,6 +189,13 @@ describe('cli tool - test option handling', function() {
       expect(outputObject.error.results[0]['message']).toEqual(
         'Every operation must have unique "operationId".'
       );
+
+      // json output should comply with written schema
+      const outputSchemaPath =
+        __dirname + '/../../../src/schemas/results-object.yaml';
+      const outputSchema = await readYaml(outputSchemaPath);
+      const results = validate(outputObject, outputSchema);
```

Comment:
Aha! so this is why you needed to make updates to results-object.yaml :joy:

Reply:
Haha yep, exposed all of the mistakes I made while writing it the first time 😂